docs/architecture/adr/0017-watchOS-use-swift.md

```diff
@@ -0,0 +1,57 @@
+---
+adr: "0017"
+status: Done
+date: 2022-12-30
+tags: [clients, watchOS]
+---
+
+# 0017 - Use Swift to build watchOS app
+
+<AdrTable frontMatter={frontMatter}></AdrTable>
+
+## Context and Problem Statement
+
+In order to build a watchOS application bundled to our iOS application we need to consider which
+technology to use. The requirements would be to remain in the same tech stack if possible and to
+have a watchOS app that we can build and can be bundled into our iOS app so that they are delivered
+together (the purpose is not to build an independant watchOS app).
```

Comment:
The context and problem statement seems a bit unclear what we're trying to achieve. It touches upon wanting a similar tech stack but the ADR itself covers which tech stack to use.

I think the context is more:

We would like to ship a watchOS application bundled togheter with the regular iOS application. The watchOS application will act as a companion application and initially offer a way to view TOTP codes using only the watch.